ui/pages/swaps/awaiting-swap/view-on-ether-scan-link/view-on-ether-scan-link.js

```diff
@@ -12,6 +12,8 @@ export default function ViewOnEtherScanLink({
 }) {
   const t = useContext(I18nContext);
 
+  const blockExplorerViewAction = 'Swap';
```

Comment:
[NIT/Weakly held opinion/change my mind] while extracting constants like this is generally good, the likelihood of the object changing in the context of its location is slim to none without major refactors. I have an ever-so-slight preference of seeing the literal string value in the `t()` callsite because it is less contextual overhead to read and understand the output.

Reply:
Could we should localize all of the `blockExplorerViewAction` values?  If they're front-facing to the user, I could see that being beneficial